### PR TITLE
CDLS-02: Add auz items to common library

### PR DESCRIPTION
**Pull request by klindziukp**
## PROBLEM
- Need to move Auz items to common library
## SOLUTION
- Move items
- Update Lombok version

### CODE REVIEW

File: src/main/java/com/codeless/api/automation/entity/security/UserDto.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public class UserDto {
+
+  @Column(name = "username")
+  @NotEmpty
+  @Email
+  private String username;
+
+  @Size(max = 100, min = 5, message = "Invalid password size. Min - 5, Max - 100.")
+  @NotEmpty
+  @Column(name = "password")
```
Comment (Strunevskiy) [inline src/main/java/com/codeless/api/automation/entity/security/UserDto.java]: 1. remove `@Column` in DTO.
2. remove Dto from name
3. move dto page in backend

File: src/main/java/com/codeless/api/automation/entity/security/View.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.codeless.api.automation.entity.security;
+
+public class View {
```
Comment (Strunevskiy) [inline src/main/java/com/codeless/api/automation/entity/security/View.java]: if it is used in DTO, please move it to dto package in backend.

File: pom.xml [hunk 3]
```diff
@@ -16,20 +16,42 @@
...
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-security</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
       <version>${lombok.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```
Comment (Strunevskiy) [inline pom.xml]: please use a lib which is specification instead of implementation for validator

File: pom.xml [hunk 4]
```diff
@@ -16,20 +16,42 @@
...
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-validator</artifactId>
+      <version>${hibernate-validator.version}</version>
+    </dependency>
```
Comment (Strunevskiy) [inline pom.xml]: if DTO is moved to backend, then remove jackson

Comment (klindziukp) [reply pom.xml]: removed

File: pom.xml (hunk 3 above)
Comment (klindziukp) [reply pom.xml]: specification used

File: src/main/java/com/codeless/api/automation/entity/security/View.java (hunk 2 above)
Comment (klindziukp) [reply src/main/java/com/codeless/api/automation/entity/security/View.java]: moved to backend

File: src/main/java/com/codeless/api/automation/entity/security/UserDto.java (hunk 1 above)
Comment (klindziukp) [reply src/main/java/com/codeless/api/automation/entity/security/UserDto.java]: renamed to AppUser and moved to Dto

**Review by Strunevskiy — APPROVED** [review]

**Pull request merged**